algorithmic_efficiency/workloads/librispeech_conformer/workload.py

```diff
@@ -100,17 +101,36 @@ def _build_dataset(self,
     elif split == 'test':
       split = 'test-clean'
 
-    ds = input_pipeline.get_librispeech_dataset(split,
-                                                data_dir,
-                                                data_rng,
-                                                train,
-                                                global_batch_size,
-                                                num_batches)
-
-    for batch in iter(ds):
-      batch = data_utils.shard_and_maybe_pad_np(
-          batch, padding_value=1, global_batch_size=global_batch_size)
-      yield batch
+    ds = LibriSpeechDataset(split=split, data_dir=data_dir)
+    ds_iter_batch_size = global_batch_size
+    sampler = None
+
+    dataloader = torch.utils.data.DataLoader(
+        ds,
+        batch_size=ds_iter_batch_size,
+        shuffle=train,
+        sampler=sampler,
+        num_workers=4,
+        prefetch_factor=10,
+        pin_memory=False,
+        drop_last=train,
+    )
+
+    dataloader = data_utils.cycle(
```

Comment:
given that the kwargs default to false, can you just make this `data_utils.cycle(dataloader)` ?